app/services/benefit_check_service.rb

```diff
@@ -42,6 +42,12 @@ def query_proxy_api
     log_error(e.message, 'Unspecified error')
   end
 
+  def query_proxy_api
+    @response = JSON.parse(RestClient.post("#{ENV['DWP_API_PROXY']}/api/benefit_checks", params))
```

Comment:
What about if the service has a problem and returns something you are not expecting ?  Is it worth checking the response HTTP status first - if it has returned anything abnormal its pointless trying to get JSON to parse it

Reply:
In the process_proxy_api_call is rescue block that will catch the error thrown by RestClient so it will not get to parsing JSON.